src/components/NavigationItem/__tests__/navigation-item.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import NavigationItem from '..';
+
+describe('NavigationItem component', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <NavigationItem
+          key="123"
+          isRead
+          isActive
+          slug="versioning"
+          title="Versioning"
+          onClick={console.log}
```

Comment:
create a noop function and remove the console.log
`function noop(): void {}`
then you can do `onClick={noop}`